### DISCUSSION TITLE
Screenshots in the directory

### DISCUSSION THREAD

**Discussion by eriknson** [MetaMask/snaps#2095] [Ideas]
Gm builders,

We're [currently scoping](https://github.com/MetaMask/snaps-directory/issues/283) the ability for developers to add screenshots to a snap's profile page in [the directory](https://snaps.metamask.io/). We believe this enhancement will improve user understanding of a snap's functionality prior to installation.

I want to hear your thoughts on this and let us know if there's anything specific you would like to see incorporated.

**Spec**
- 3 images (no less, no more)
- stored in the registry
- 16:9 aspect ratio
- exactly 960px width, 540px height

**Example**
![image](https://github.com/MetaMask/snaps/assets/34306844/93179b9e-d105-4755-8250-5e072146c53c)
![image](https://github.com/MetaMask/snaps/assets/34306844/49ce39f9-3f5f-4aaf-aae5-13e23378e78c)

**Comment by cryptoKevinL** [comment]
Just curious, why the requirement for wide aspect ratio of that size when Snaps are actually the opposite?
